README.md

```diff
@@ -1,12 +1,10 @@
 # react-a11y-announcer
 
-Welcome to React Announcer from Think Company.
-
-This is a React component that demonstrates how to dynamically add announcements to a web page, and ensure that they are accessible and announced properly by screen readers.
+`react-a11y-announcer` is a React a11y component that helps ensure announcements are accessible and properly announced by screen readers. 
```

Comment:
That's a very good point and something to consider going forward, not everyone immediately knows what certain acronyms are, so we should always link the first instance of it. If we can opt to use the full word, we should do that too.